app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb

```diff
@@ -87,7 +87,7 @@
   </section>
 
   <section class="col col-720 col-space-m">
-    <h2 class="heading-s">How we can help</p>
+    <h2>How we can help</h2>
 
     <p>If you would like more information about the course and application process, you can talk to a teacher training adviser for free. They can help you to:</p>
```

Comment:
Do you think we should direct people to advisers at this point? As we've got the CTA at the end of the other pages.